spec/std/static_array_spec.cr

```diff
@@ -90,6 +90,25 @@ describe "StaticArray" do
     end
   end
 
+  it "shuffles with a seed" do
+      a = StaticArray(Int32, 10).new { |i| i + 1 }
```

Comment:
Please use 2 spaces for indentation, or just run `crystal tool format` to fix this.